src/tangle/sources/mod.rs

```diff
@@ -28,14 +72,14 @@ pub trait InputSource {
     /// Retrieves a stream of milestones and their protocol parameters given a range of indexes.
     async fn milestone_stream(
         &self,
-        range: std::ops::Range<MilestoneIndex>,
+        range: MilestoneRange,
     ) -> Result<BoxStream<Result<MilestoneAndProtocolParameters, Self::Error>>, Self::Error>;
 
     /// Retrieves a stream of blocks and their metadata in white-flag order given a milestone index.
     async fn cone_stream(
         &self,
         index: MilestoneIndex,
-    ) -> /* TODO: This should not require enriching the inputs already */ Result<BoxStream<Result<BlockWithMetadataInputs, Self::Error>>, Self::Error>;
```

Comment:
That `TODO` should probably stay there.

Reply:
It got removed automagically, but i re-added it.